api-reference/20 Data Visualization Widgets/dxSankey/1 Configuration/alignment.md

```diff
@@ -43,6 +43,44 @@ A string value aligns all node columns uniformly. An array of strings allows you
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxSankey ... 
+            :alignment="['top', 'bottom', 'bottom']">
+        </DxSankey>
+    </template>
+
+    <script>
+    import DxSankey from 'devextreme-vue/sankey';
+
+    export default {
+        components: {
+            DxSankey
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import Sankey from 'devextreme-react/sankey';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Sankey ... 
+                    alignment={['top', 'bottom', 'bottom']}>
```

Comment:
In React, declare arrays as a component property or a constant/variable before the class declaration.